iris-mpc-cpu/src/hnsw/graph/graph_store.rs

```diff
@@ -15,7 +15,11 @@ use sqlx::{
     types::{Json, Text},
     Executor, Postgres, Row, Transaction,
 };
-use std::{marker::PhantomData, str::FromStr};
+use std::{
+    marker::PhantomData,
+    ops::{Deref, DerefMut},
```

Comment:
`clippy` complains on this unused `Deref`

Reply:
Fixed.